Dockerfile

```diff
@@ -15,15 +15,15 @@ RUN git clone https://github.com/swagger-api/swagger-ui.git /tmp/swagger-ui \
     && rm -rf /tmp/swagger-ui \
     && cp /go/src/github.com/cyverse-de/permissions/index.html /docs/index.html
 
-RUN go get github.com/constabulary/gb/...
-
-RUN cd /go/src/github.com/cyverse-de/permissions && \
-    gb build && \
-    cp bin/permissions-server /bin/permissions
+RUN go install github.com/cyverse-de/permissions/... \
+    && cp /go/bin/permissions-server /bin/permissions
```

Comment:
Heh, it wasn't really that easy. I also had to move a bunch of files around and change the imports to refer to the full dependency name.